pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -454,9 +455,28 @@ class Utils {
     @Restricted(NoExternalUse.class)
     static <T> T instantiateDescribable(Class<T> c, Map<String, ?> args) {
         DescribableModel<T> model = new DescribableModel<>(c)
+        // Special case for JENKINS-63499.
+        if (model != null && model.type == PasswordParameterDefinition.class && model.getParameter("defaultValueAsSecret") != null) {
```

Comment:
`model.getParameter("defaultValueAsSecret") != null` makes sure that we only run this code when running Jenkins 2.236 or newer.